package.json

```diff
@@ -2,6 +2,7 @@
   "name": "antd-demo",
   "version": "0.1.0",
   "private": true,
+  "proxy" : "http://localhost:3000",
```

Comment:
are you committing as `localhost`?

Reply:
actually this line isn't needed, it was a fix ronak tried to fix the cors issue but it makes no difference